### PR TITLE
WIP init work on collections

### PR DESCRIPTION
**Pull request by matthewtrask**
dont merge

### CODE REVIEW

**Comment by MikeRalphson** [comment]
:hooray:

File: _config.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+site:
+  url: https://openapi.tools
+description: A site to find the right tools for developers to build with the OpenAPI  
+collections_dir: /reviews
```
Comment (MikeRalphson) [inline _config.yml]: Consider changing to something like `content` to allow different types of collections....

File: _config.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+collections:
+  reviews:
+    name: OAI Tools
+    output: true
+    permalink: /reviews/:tool
```
Comment (MikeRalphson) [inline _config.yml]: You can remove this line, it's done automatically.

File: _config.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+    relative_directory: /reviews
```
Comment (MikeRalphson) [inline _config.yml]: Maybe not needed?

File: _data/reviews.yml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# - name: Dredd
```
Comment (MikeRalphson) [inline _data/reviews.yml]: Whole file can be removed. Collections aren't under data.

**Comment by MikeRalphson** [comment]
Then chuck in a file called `dredd.md` with yaml preface in `content/reviews` and it should hopefully just work!

**Comment by matthewtrask** [comment]
@philsturgeon @MikeRalphson still working on getting the page to render. Once I get that, I think its pretty much good.

**Pull request closed without merging**